libc/newhdrgen/tests/output/test_small.h

```diff
@@ -28,20 +28,20 @@ enum {
 
 __BEGIN_C_DECLS
 
-CONST_FUNC_A void func_a() __NOEXCEPT;
+CONST_FUNC_A void func_a()__NOEXCEPT;
```

Comment:
I think this should have spaces still

Reply:
sounds good, once the cmake patch goes up, I'll make an edit to the header.py for spacing.